scripts/scr_battle_roster/scr_battle_roster.gml

```diff
@@ -42,32 +42,40 @@ function scr_battle_roster(_battle_loci, _loci_specific, _is_planet) {
 
 
 	// if (_is_planet=true){
-	    var company, v, meat;
+	    var company, v, meat, marine;
```

Comment:
v?

Reply:
Refactored both for the next commit. v it's just an index for the company array. meat is related to obj_temp_meeting, which, at least for now, im not too familiar with (probably some special case, as with the spyrer fight).